### ISSUE TITLE
False positive "The closure can be replaced with" for arrow function

### ISSUE THREAD

**Issue by Gruven** [kalessil/phpinspectionsea#1624]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Ultimate) |
| Language level | PHP 7.4                        |

Current behaviour
--
For this kind of code
```php
$okayArray = array_map(
  fn($element) => Element::isOkay($element),
  $elements,
);
```
EA says
> [EA] The closure can be replaced with 'isOkay' (reduces cognitive load). 

and fixes it to
```php
$okayArray = array_map(
  'isOkay',
  $elements,
);
```
For static anonymous function it doesn't say anything
```php
$okayArray = array_map(
  static function ($element) {
    return Element::isOkay($element);
  },
  $elements,
);
```

Expected behaviour
--
No warning

Environment details
---
PhpStorm 2020.2.1
Build #PS-202.6948.87, built on August 27, 2020
Runtime version: 11.0.8+10-b944.31 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Linux 5.4.0-42-generic
GC: ParNew, ConcurrentMarkSweep
Memory: 8108M
Cores: 12
Registry: run.processes.with.pty=TRUE, ide.balloon.shadow.size=0, ide.suppress.double.click.handler=true
Non-Bundled Plugins: String Manipulation, com.bluewaitor.tsReact, com.chrisrm.idea.MaterialThemeUI, com.intellij.ideolog, org.intellij.plugins.postcss, com.mallowigi, net.seesharpsoft.intellij.plugins.csv, intellij.prettierJS, com.dmarcotte.handlebars, de.espend.idea.laravel, com.kalessil.phpStorm.phpInspectionsUltimate, de.espend.idea.php.toolbox, de.espend.idea.php.annotation, izhangzhihao.rainbow.brackets, ru.adelf.idea.dotenv
Current Desktop: X-Cinnamon

**Comment by jdreesen** [comment]
Duplicate of #1608 which should be fixed in the next release.
